src/main/java/com/danigu/blog/common/TwoWayTransformer.java

```diff
@@ -0,0 +1,10 @@
+package com.danigu.blog.common;
+
+/**
+ * Interface for two-way transformer.
+ * @param <F> from
+ * @param <T> to
+ */
+public interface TwoWayTransformer<F, T> extends OneWayTransformer<F, T> {
```

Comment:
I think using the simply "Transformer" is sufficient. I would recommend to add fromEntity and toEntity methods.